accio-tests/src/main/java/io/accio/testing/TestingWireProtocolClient.java

```diff
@@ -1129,6 +1131,22 @@ public FormatCodes.FormatCode getFormatCode()
         public byte[] getFormattedBytes()
         {
             if (formatCode == TEXT) {
+                if (type instanceof PGArray) {
+                    StringBuilder stringBuilder = new StringBuilder();
+                    stringBuilder.append("{");
+                    for (Object element : (Object[]) value) {
+                        if (element == null) {
+                            stringBuilder.append("NULL");
+                        }
+                        else {
+                            stringBuilder.append(element);
+                        }
+                        stringBuilder.append(',');
+                    }
+                    stringBuilder.setLength(stringBuilder.length() - 1);
+                    stringBuilder.append("}");
+                    return stringBuilder.toString().getBytes(UTF_8);
```

Comment:
Pardon, is this another pg jdbc behavior or something that was defined in pg wire protocol ? Not sure why the PGArray format is `{object,object,object}`

Reply:
https://github.com/Canner/accio/blob/80e3bcfb5c7a9843aaed128564782f18ceff6749/accio-base/src/main/java/io/accio/base/type/PGArray.java#L280-L283

It's the definition in PG Wire Protocol.